text/0086-keptn-lts.md

```diff
@@ -0,0 +1,220 @@
+
+
+### Support timeline and policy
+
+
+* Keptn LTS versions will be supported for at least 6 months after the initial release.
+  The Keptn community decides when to stop the support
+    * Keptn contributors and users can be extended beyond 6 months, as long as community members (maintainers, end users or vendors) are willing to provide support.
+    * 6 months timeline is chosen initially to evaluate LTS and its adoption. The minimum support term may be extended later based on the evaluation
+* Keptn community announces the LTS baseline end-of-life with 3 months advance, 
+  using the community announcement channels
+ (#announcements on Slack is considered the official one)
+* If Keptn LTS includes preview features, they are not covered by the LTS support policy.
+  Breaking changes might be introduced there even within LTS,
+  and the preview features can be also removed altogether should it be needed for better LTS stability.
+
+### Release cadence 
+
+We are not sure about the adoption of the LTS release and the cadence of the upcoming major changes in Keptn components.
+Based on that, the approach in this KEP makes no commitment on release cadence:
+
+* Keptn community may release new LTS baselines based on the decision by the community.
+  Any community member can suggest making a new LTS release baseline.
+* For patch releases, the Keptn community intends releasing a new version with backports
+  when there are major bug fixes that need to be backported.
+  Any community member can suggest making a new LTS patch release, and the decision will be made by Keptn community and core maintainers
+* For security issues, the [Keptn Security Process](https://github.com/keptn/.github/blob/main/SECURITY.md) will be followed for all currently supported LTS baselines.
+  Security patches for the LTS need to be released along with security patches for the main baseline.
+
+For the normal release line, we also remove monthly releases for MINOR releases with new features.
+We target releasing Keptn on a monthly basis if there is something to deliver.
+Patch releases for bugfixes and security releases will be done on-demand. 
+
+
+## Changes to be made
+
+### Keptn Specification
+
+No changes planned at the moment, except documentation.
+
+### Keptn Core
+
+Keptn core changes to be minimized to the pending stories:
+
+* Performance improvements
+* Code cleanup
+* Maintainability & Supportability
+* Landing open pull requests with valuable functionality
+
+Notably Keptn GitOps will remain in preview, or KEP-60 RBAC will not be shipped in the first LTS release.
+The timeline for these releases is likely to be delayed by the community.
+
+
+### Documentation changes
+
+In the LTS we want to improve maintainability of documentation and
+to land pending stories in the documentation working group.
+
+
+* Adopt Documentation-as-Code in Keptn where reasonable
+    * Support for distributed documentation: multiple repositories, branches and versions.
```

Comment:
We aren't going to be able to achieve this by the time LTS is released, right?  Is the plan to silently roll to this in the first few months after LTS is released?